lib/Backend/Commands.hs

```diff
@@ -91,7 +91,19 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
```

Comment:
Let's add some haddocks here explaining what/why we're doing this:

```suggestion
    -- | When attempting to create an entry at, e.g., @/a/b/c@, the directories
    -- @/a@ and @/a/b@ will be implicitly created. 
    -- 
    -- This function checks whether those paths are already
    -- occupied by existing entries and, if so, fails.
    --
    -- Note: the root path @/@ cannot possibly be occupied by an entry, so therefore
    -- we skip the check for that path.
    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
```

Reply:
Done